workers/generic-worker/mounts.go

```diff
@@ -467,6 +467,10 @@ func (taskMount *TaskMount) Stop(err *ExecutionErrors) {
 			err.add(Failure(e))
 		}
 	}
+	err.add(executionError(internalError, errored, fileutil.WriteToFileAsJSON(&fileCaches, "file-caches.json")))
```

Comment:
https://github.com/taskcluster/taskcluster/blob/8cad35306ffe8ecb8442b5fcd7af567ee1eef676/workers/generic-worker/mounts.go#L150 shouldn't this get removed too then?

Reply:
Faster than our CI 😉

Reply:
whoops, thought i had deleted that 😬 
thanks!